src/Microsoft.DotNet.Darc/Darc/Operations/UpdateDependenciesOperation.cs

```diff
@@ -87,7 +87,7 @@ public override async Task<int> ExecuteAsync()
                 }
                 catch (DarcException exc)
                 {
-                    Logger.LogError(exc, $"Error: Failed to update dependencies based on folder '{_options.PackagesFolder}'");
+                    Logger.LogError(exc, $"Failed to update dependencies based on folder '{_options.PackagesFolder}'");
```

Comment:
Can we use structural logging here? We have an open issue to add Application Insights to darc. If we ever do it will be useful